synthesizer/program/src/logic/command/get_or_use.rs

```diff
@@ -14,21 +14,22 @@
 
 use crate::{
     traits::{FinalizeStoreTrait, RegistersLoad, RegistersStore, StackMatches, StackProgram},
+    CallOperator,
     Opcode,
     Operand,
 };
 use console::{
     network::prelude::*,
-    program::{Identifier, Register, Value},
+    program::{Register, Value},
 };
 
 /// A get command that uses the provided default in case of failure, e.g. `get.or_use accounts[r0] r1 into r2;`.
 /// Gets the value stored at `operand` in `mapping` and stores the result in `destination`.
 /// If the key is not present, `default` is stored in `destination`.
 #[derive(Clone, PartialEq, Eq, Hash)]
 pub struct GetOrUse<N: Network> {
-    /// The mapping name.
-    mapping: Identifier<N>,
+    /// The mapping.
+    mapping: CallOperator<N>,
```

Comment:
Same as above. 

This will break the serialization of programs on the current network.

Current programs with the `Identifier` mapping will fail to deserialize because `CallOperator::Resource` requires an extra `variant` byte.

Reply:
Fixed with new serialization method.